docs/reference/dependencies.md

```diff
@@ -1061,249 +1065,266 @@ All of these libraries are required for core functionality.
-- spring-boot-starter-logging @2.7.2, [(Apache License, Version 2.0)](https://www.apache.org/licenses/LICENSE-2.0)
-- spring-boot-starter-tomcat @2.7.2, [(Apache License, Version 2.0)](https://www.apache.org/licenses/LICENSE-2.0)
-- spring-boot-starter-validation @2.7.2, [(Apache License, Version 2.0)](https://www.apache.org/licenses/LICENSE-2.0)
-- spring-boot-starter-web @2.7.2, [(Apache License, Version 2.0)](https://www.apache.org/licenses/LICENSE-2.0)
-- spring-boot-starter @2.7.2, [(Apache License, Version 2.0)](https://www.apache.org/licenses/LICENSE-2.0)
-- spring-boot @2.7.2, [(Apache License, Version 2.0)](https://www.apache.org/licenses/LICENSE-2.0)
-- spring-security-config @5.7.2, [(Apache License, Version 2.0)](https://www.apache.org/licenses/LICENSE-2.0)
-- spring-security-core @5.7.2, [(Apache License, Version 2.0)](https://www.apache.org/licenses/LICENSE-2.0)
-- spring-security-crypto @5.7.2, [(Apache License, Version 2.0)](https://www.apache.org/licenses/LICENSE-2.0)
-- spring-security-oauth2-client @5.7.2, [(Apache License, Version 2.0)](https://www.apache.org/licenses/LICENSE-2.0)
-- spring-security-oauth2-core @5.7.2, [(Apache License, Version 2.0)](https://www.apache.org/licenses/LICENSE-2.0)
-- spring-security-oauth2-jose @5.7.2, [(Apache License, Version 2.0)](https://www.apache.org/licenses/LICENSE-2.0)
-- spring-security-oauth2-resource-server @5.7.2, [(Apache License, Version 2.0)](https://www.apache.org/licenses/LICENSE-2.0)
-- spring-security-web @5.7.2, [(Apache License, Version 2.0)](https://www.apache.org/licenses/LICENSE-2.0)
-- spring-aop @5.3.22, [(Apache License, Version 2.0)](https://www.apache.org/licenses/LICENSE-2.0)
-- spring-beans @5.3.22, [(Apache License, Version 2.0)](https://www.apache.org/licenses/LICENSE-2.0)
-- spring-context-support @5.3.22, [(Apache License, Version 2.0)](https://www.apache.org/licenses/LICENSE-2.0)
-- spring-context @5.3.22, [(Apache License, Version 2.0)](https://www.apache.org/licenses/LICENSE-2.0)
-- spring-core @5.3.22, [(Apache License, Version 2.0)](https://www.apache.org/licenses/LICENSE-2.0)
-- spring-expression @5.3.22, [(Apache License, Version 2.0)](https://www.apache.org/licenses/LICENSE-2.0)
-- spring-jcl @5.3.22, [(Apache License, Version 2.0)](https://www.apache.org/licenses/LICENSE-2.0)
-- spring-tx @5.3.22, [(Apache License, Version 2.0)](https://www.apache.org/licenses/LICENSE-2.0)
-- spring-web @5.3.22, [(Apache License, Version 2.0)](https://www.apache.org/licenses/LICENSE-2.0)
-- spring-webmvc @5.3.22, [(Apache License, Version 2.0)](https://www.apache.org/licenses/LICENSE-2.0)
-- spring-websocket @5.3.22, [(Apache License, Version 2.0)](https://www.apache.org/licenses/LICENSE-2.0)
-- testcontainers @1.17.1, [(MIT)](http://opensource.org/licenses/MIT)
-- snakeyaml @1.30, [(Apache License, Version 2.0)](http://www.apache.org/licenses/LICENSE-2.0.txt)
+- logback-classic@1.2.11, [(Eclipse Public License - v 1.0)](http://www.eclipse.org/legal/epl-v10.html)
```

Comment:
these will render as `mailto` links if we  don't separate the `@` as before

Reply:
Remember this for next time, but I'll cover it today 😃